deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -133,10 +131,8 @@ spec:
           - apiGroups:
               - apps
             resources:
-              - deployments
```

Comment:
@rigazilla this should be aligned with role.yaml

Reply:
yeah sure

Reply:
mmm, actually the `permissions` is generated by the operator-sdk in the right place from role.yaml.
I would remove it from here so we don't have to align duplicated text